runtime/compiler/optimizer/SPMDPreCheck.cpp

```diff
@@ -29,11 +29,13 @@
 
 bool SPMDPreCheck::isSPMDCandidate(TR::Compilation *comp, TR_RegionStructure *loop)
    {
- 
+   bool trace = comp->getOption(TR_TraceAll);
```

Comment:
Shoudl this be `comp->getOption(TR_TraceALL) || trace()` to make sure the trace turns on if only SPMDPreCheck is being traced?